contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
+
+## Which trailers are to be added?
+
+- `Helped-by: Full Name <email@address>`
+   Added by regular users to particular commits within a PR. Applies to the commit it appears on. Means "this commit was somehow improved based on suggestions by @reviewer".
+- `Reviewed-by: Full Name <email@address>`
+   Added by maintainer to PR merge commit. Applies to whole PR. Means "these changes were carefully reviewed by @reviewer an he has stated conformity with them."
```

Comment:
Ok. Regarding gender-neutrality, well, I'm ok with it too. When I learned English, I was taught that "generic he" was the correct thing. Nowadays, though, it seems "singular they" is having more adepts.I don't have a strong position on that. I just think it's a bit of a pity some people make an argument out of that. To me, offense is in the intention, not in the words you use. But, again, I don't have any problem in using singular they, if that's considered to be better here.